packages/manager/modules/pci/src/projects/new/error/error.module.js

```diff
@@ -0,0 +1,38 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+import '@ovh-ux/manager-error-page';
+
+import './error.scss';
```

Comment:
Imports are a bit mixed between third party modules and local style files.

Do you think they could be grouped?